### DISCUSSION TITLE
Help with R Document Issue in VSCode (Radian works in terminal but fails in VSCode)

### DISCUSSION THREAD

**Discussion by apolveiro** [microsoft/vscode-discussions#2386] [Extension Development QnA] [answered]
Hello everyone,

I'm encountering an issue when trying to create a new R document in VSCode. While radian works fine in the terminal, when I try to open a new file as an "R Document" in VSCode, I get the following error:

Error Message:

Failed to run diagnostics: ! in callr subprocess.
Caused by error: 
! Linter 'object_length_linter' failed in : path[1]="" O sistema não pode encontrar o caminho especificado

The terminal output when running radian in PowerShell looks like this:

PS C:\Users\Arthur Polveiro> radian
R version 4.4.2 (2024-10-31 ucrt) -- "Pile of Leaves"
Platform: x86_64-w64-mingw32 (64-bit)

r$>

So, R and radian seem to be working fine in the terminal, but I can't create a new R document in VSCode without encountering this error.

I've tried:

Verifying the paths in VSCode settings
Reinstalling R and radian
Checking for any issues with the lintr package
Would appreciate any insights or suggestions on how to resolve this. Thanks!

**Comment by alefragnani** [comment, marked as answer]
Hi @apolveiro ,

I don’t know R, nor Radian, so I would ask you, are you following this documentation (https://code.visualstudio.com/docs/languages/r) to use R in VS Code. 

If that’s the case, I would suggest you to take a look in the repo of the R extension (https://github.com/REditorSupport/vscode-R/issues) for issues and help on configuring Radian on VS Code. A Quick Look there, and I could find this issue https://github.com/REditorSupport/vscode-R/issues/1419.  If you are using another extension or tutorial, look in the proper extension repo.

This discussions repo is intended for extension development only, so questions about using VS Code or some specific extension, should be redirected to the proper repo, either VS Code or the extension.

Hope this helps
